presto-orc/src/main/java/com/facebook/presto/orc/OrcWriteValidation.java

```diff
@@ -830,6 +970,8 @@ public long getHash()
         private final Map<Long, StripeStatistics> stripeStatistics = new HashMap<>();
         private List<ColumnStatistics> fileStatistics;
         private long retainedSize = INSTANCE_SIZE;
+        private Set<Integer> flattenedNodes;
```

Comment:
I needed a few more fields passed from the writer to build those collections of various flat map nodes,